Daemon/Daemon/Monitor.m

```diff
@@ -531,7 +543,32 @@ -(void)processEvent:(File*)file
     
     //dbg msg
     logMsg(LOG_DEBUG, [NSString stringWithFormat:@"created event: %@", event]);
-    
+
+    //matches last event?
+    // if so, ignore the event
+    if(YES == [event isRelated:self.lastEvent])
```

Comment:
Before matching rules against the given event, we can exit earlier if the event is the same as the last processed event.